data/pug/pug news.txt

```diff
@@ -0,0 +1,363 @@
+
+news "artist on post-fw Deneb"
+	location
+		government Neutral
+		system Deneb
+	name
+		word
+			"Writer"
+			"Painter"
+			"Artist"
+			"Sculptor"
+			"Designer"
+			"Photographer"
+			"Composer"
+	message
+		word
+			"This "
+		word
+			"planet"
+			"city"
+			"building"
+		word
+			"'s "
+		word
+			"alien"
+			"extraterrestrial"
+			"unique"
+			"Pug"
+		word
+			" origin is so "
+		word
+			"inspiring"
+			"intriguing"
+			"thrilling"
+			"exciting"
+		word
+			"! I'm "
+		word
+			"full of"
+			"bristling with"
+		word
+			" "
+		word
+			"ideas"
+			"creativity"
+			"energy"
+		word
+			"!"
+
+news "tourist on post-fw Deneb"
+	location
+		government Neutral
+		system Deneb
+	name
+		word
+			"Tourist"
+			"Sightseer"
+			"Traveler"
+	message
+		word
+			"Can you imagine"
+			"Did you know"
+		word
+			" that "
+		word
+			"this entire"
+			"this whole"
+			"all this"
+		word
+			" "
+		word
+			"city"
+			"building"
+		word
+			" was "
+		word
+			"constructed"
+			"created"
+			"built"
+			"made"
+		word
+			" by "
+		word
+			"an alien species"
+			"aliens"
+			"insectoid aliens"
+			"another race"
+		word
+			"? "
+		word
+			"That's"
+			"I find that"
+		word
+			" "
+			" so "
+		word
+			"inspiring"
+			"incredible"
+			"exciting"
+			"fascinating"
+		word
+			"!"
+
+news "conspiracy theorist on post-fw Deneb"
+	location
+		government Neutral
+		system Deneb
+	name
+		word
+			"Conspiracy Theorist"
+			"Critical Thinker"
+			"Truth Seeker"
+	message
+		word
+			"Do you really"
+			"How can you"
+		word
+			" "
+		word
+			"believe"
+			"still think"
+			"accept"
+			"trust"
+		word
+			" that this "
+		word
+			"city"
+			"spaceport"
+			"building"
+			"place"
+		word
+			" was "
+		word
+			"created"
+			"made"
+			"inhabited"
+		word
+			" by "
+		word
+			"another species"
+			"aliens"
+			"a different race"
+		word
+			"? "
+		word
+			"This was "
+		word
+			"clearly"
+			"obviously"
+			"without a doubt"
+		word
+			" built for humans, by humans. "
+		word
+			"You will eventually realize that"
+			"You will come to see that"
+			"I'm gathering evidence for a public trial"
+			"I can provide you with all the proof you need"
+			"This is all just a stunt by the Republic and the Free Worlds"
+		word
+			"."
+
+news "reporter on post-fw Deneb"
+	location
+		government Neutral
+		system Deneb
+	name
+		word
+			"Reporter"
+			"Journalist"
+			"Correspondent"
+	message
+		word
+			"I don't think there is anything"
+			"I don't find anything to be"
+			"There isn't anything"
+		word
+			" "
+		word
+			"special"
+			"unique"
+			"different"
+		word
+			" about this "
+		word
+			"planet"
+			"place"
+		word
+			", but "
+		word
+			"the people"
+			"my viewers"
+		word
+			" "
+		word
+			"believe otherwise"
+			"always love when I report from here"
+			"still love to see me here"
+			"like the sound of an alien history"
+			"find it fascinating"
+		word
+			". Anyway, I "
+		word
+			"have to go now"
+			"'m off to the studio"
```

Comment:
```suggestion
			". Anyway, I"
		word
			" have to go now"
			"'m off to the studio"
```
Potential solution to [above review](https://github.com/endless-sky/endless-sky/pull/8911/files#r1233103850).